config/misc.go

```diff
@@ -35,19 +35,15 @@ func GetInstanceID() string {
 	instance := GetString("INSTANCE_ID", "")
 	instanceArr := strings.Split(instance, "-")
 	length := len(instanceArr)
+	regexGwHa := regexp.MustCompile(`^.*-gw-ha-\d+-\w+-\w+$`)
+	regexGwNonHaOrProcessor := regexp.MustCompile(`^.*-\d+$`)
```

Comment:
We can define those regexes as a global vars in the file, so we don't have to compile everytime `GetInstanceID` is called